cmd/collector/app/handler/grpc_handler.go

```diff
@@ -21,6 +21,9 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	// enable gzip compression support
+	_ "google.golang.org/grpc/encoding/gzip"
```

Comment:
this won't work, run `make fmt`. The comment isn't necessary, ".../encoding/gzip" hints at that already

Reply:
This comment is required by golint.

```
cmd/collector/app/handler/grpc_handler.go:22:2: a blank import should be only in a main or test package, or have a comment justifying it
```